layouts/partials/webinar-list-item.html

```diff
@@ -0,0 +1,21 @@
+<li class="w-full md:w-1/2 lg:w-1/3 m-0 p-2">
+    <article class="rounded shadow-md bg-white border border-gray-200 mb-10 flex flex-col mx-auto" style="max-width: 320px;">
+        <div style="height: 239px;">
+            <div class="w-full h-full bg-contain bg-no-repeat bg-center bg-purple-500" style="background-image: url('{{ .Params.preview_image }}');"></div>
+        </div>
+        <div class="h-32 p-3">
+            <h3>
+                {{ if .Params.external }}
+                    <a class="text-2xl" target="_blank" rel="noopener noreferrer" href="{{ .Params.url_slug }}">
+                        {{ .Params.title }}
+                        <i class="text-sm ml-2 fas fa-external-link-alt"></i>
+                    </a>
+                {{ else }}
+                    <a class="text-2xl" href="/webinar/{{ .Params.url_slug }}/">
+                        {{ .Params.title }}
```

Comment:
This is overflowing for the CrossGuard webinar:

![image](https://user-images.githubusercontent.com/710598/76917484-c47f4880-6880-11ea-920d-a5fff5d81786.png)

http://content-bucket-1908485.s3-website-us-east-1.amazonaws.com/webinar/

Reply:
Good catch. I ended up having to rework the background a bit and give everything a black background so the different images "look" the same.